pyds8k/resources/ds8k/v1/cs/flashcopies.py

```diff
@@ -0,0 +1,68 @@
+##############################################################################
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##############################################################################
+
+"""
+FlashCopies interface.
+"""
+from pyds8k.base import ManagerMeta, ResourceMeta
+from ..common.base import Base, BaseManager
+from ..common.types import DS8K_FLASHCOPIES, DS8K_FLASHCOPY
+from ..volumes import Volume, VolumeManager
+
+
+class FlashCopy(Base, metaclass=ResourceMeta):
+    resource_type = DS8K_FLASHCOPIES
+    _template = {'id': None,
+                 'persistent': None,
+                 'recording': None,
+                 'backgroundcopy': None,
+                 'state': None,
+                 'options': [],
+                 'volume_pairs': []
+                 }
+
+    related_resource = {'sourcevolume': (Volume, VolumeManager),
+                        'targetvolume': (Volume, VolumeManager)
+                        }
+
+    def __repr__(self):
+        return "<FlashCopy: {0}>".format(self._get_id())
+
+    def _add_details(self, info, force=False):
+        super(FlashCopy, self)._add_details(info, force=force)
+        if info.get(DS8K_FLASHCOPY):
+            self._id = info[DS8K_FLASHCOPY][0]['id']
+
+
+class FlashCopyManager(BaseManager, metaclass=ManagerMeta):
+    """
+    Manage FlashCopies resources.
+    """
+    resource_class = FlashCopy
+    resource_type = DS8K_FLASHCOPIES
+
+    def get(self, resource_id='', url='', obj_class=None, **kwargs):
+        return self._get(resource_id=resource_id, url=url,
+                         obj_class=obj_class, **kwargs)
+
+    def list(self, url='', obj_class=None, body=None, **kwargs):
+        return self._list(url=url, obj_class=obj_class, body=body, **kwargs)
+
+    def posta(self, url='', body=None):
+        return self._posta(url=url, body=body)
+
```

Comment:
do we know if we also need `put`/`patch`?

Reply:
I don't know, and there is only PUT in the api guide for update

Reply:
then should we add `put`?

Reply:
well, in second checking, there is no put ether as is specify in the [user guide](https://www.ibm.com/support/pages/sites/default/files/inline-files/$FILE/sc27923502.pdf)

Reply:
great